Tests/SwiftDocCTests/Infrastructure/DocumentationCuratorTests.swift

```diff
@@ -279,6 +279,10 @@ class DocumentationCuratorTests: XCTestCase {
             ### Extraneous list item content
             - <doc:MyKit/MyClass>.
             - <doc:MyKit/MyClass> ![](featured.png) and *more* content...
+            - <doc:MyKit/MyClass> @Comment { This is an invalid comment directive }
+            - <doc:MyKit/MyClass>   <!-- This is a valid comment -->
+            - <doc:MyKit/MyClass> <!-- This is a valid comment --> but this is extra content :(
+            - <doc:MyKit/MyClass> This is extra content <!-- even if this is a valid comment -->
```

Comment:
Considering the `child is InlineHTML` check; what would the behavior be for this?
```
- <doc:MyKit/MyClass> <strong>This is extra content</strong>
```

Reply:
There would be no warning. Is it assumed that any HTML used will be a comment? It looked like HTMLBlocks were assumed to be comments elsewhere, like in abstracts.